src/CftfBundle/Entity/LsItem.php

```diff
@@ -946,6 +946,17 @@ function (LsItem $item) {
         return $ids->toArray();
     }
 
+    /*
+     * Find all children items of selected item
+     */
+    public function getDescendantIds($childIds)
```

Comment:
- This still only gets child ids, not ids for all descendants
- `$childIds` is probably not needed as an argument
- There is no return, so I am assuming things are not working right now at all.